app/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.test.tsx

```diff
@@ -374,6 +374,16 @@ describe("FormAudience", () => {
       });
       expect(screen.getByTestId("isSticky")).toBeChecked();
     }
+
+    await act(async () => {
+      const field = within(
+        screen.queryByTestId("population-percent-top-row") as HTMLElement,
+      ).getByTestId("population-percent-input");
+      fireEvent.click(field);
+      fireEvent.change(field, { target: { value: "123" } });
```

Comment:
Nitty-gritty: percentage 123? I think we should restrict to max  100?

Reply:
Copy/paste, my nemesis! Good catch 🔥